geth/whisper/request_historic_messages.go

```diff
@@ -165,3 +160,27 @@ func makePayloadData(r historicMessagesRequest) []byte {
 	copy(data[8:], r.Topic[:])
 	return data
 }
+
+//getPeerID is used to get peerID from string values of peerID or enode
+func getPeerID(m map[string]interface{}) ([]byte, error) {
+	peerInterfaceValue, okPeer := m["peer"]
+	enodeInterfaceValue, okEnode := m["enode"]
+
+	if okPeer == okEnode { //only if existing peer or enode(!xor)
```

Comment:
Nice condition! Could you move comment just before `if` statement?

Reply:
done